package.json

```diff
@@ -27,6 +27,7 @@
     "bootstrap": "^4.6.0",
     "bootstrap-slider": "^10.6.2",
     "chart.js": "^2.9.4",
+    "cheerio": "^1.0.0-rc.10",
```

Comment:
This is a dependency that is only used by the docenizer script. I'm not sure if we want it here or if we want a separate package.json and node_modules root inside /etc/scripts? Another option is to move the docenizers into a new repository.

Reply:
After disussing it internally, we might want to just leave this kind of scripts here :)
Could you move this to devDependencies though? If there's any package that should be there, I think it's this one

Reply:
Moved in b0ef781dcb1c277a72f8a00d562e22ab0a781ee5